conda_smithy/ci_register.py

```diff
@@ -81,7 +81,7 @@ def add_token_to_circle(user, project):
 def add_project_to_circle(user, project):
     headers = {'Content-Type': 'application/json',
                'Accept': 'application/json'}
-    url_template = ('https://circleci.com/api/v1/{component}?'
+    url_template = ('https://circleci.com/api/v1.1/{component}?'
```

Comment:
Would go ahead and include `project/github/` before `{component}` and strip `project` from the urls below.